Source/WebCore/page/LocalDOMWindow.h

```diff
@@ -451,6 +464,17 @@ class LocalDOMWindow final
 #endif
 
     RefPtr<CookieStore> m_cookieStore;
+
+#if ENABLE(DECLARATIVE_WEB_PUSH)
+    bool isActive() const final { return true; }
```

Comment:
I don't think we should mix member functions and data members, even if it is convenient for your ifdefs. Please move the member functions earlier in the header, in their own `#if ENABLE(DECLARATIVE_WEB_PUSH)` block, with other member functions.